crates/ironrdp-cliprdr-native/src/windows/os_clipboard.rs

```diff
@@ -117,8 +106,6 @@ impl Drop for OwnedOsClipboard {
     fn drop(&mut self) {
         // SAFETY: We own the clipboard at moment of method invocation, therefore it is safe to
         // call `CloseClipboard`.
-        if unsafe { CloseClipboard() } == FALSE {
-            error!("Failed to close Windows clipboard");
-        }
+        let _ = unsafe { CloseClipboard() };
```

Comment:
question: Aren’t we interested in knowing if `CloseClipboard` failed? I see you ignored many other errors too. Just curious to know why it’s fine to ignore all of that.

Reply:
I updated code paths to print error!()